docs/source/using_the_ve/getting_started.md

```diff
@@ -56,18 +56,36 @@ configuration and data files to run a model.
 ve_run --install-example /path/
 ```
 
-You can then run the model itself:
+You can then run the model itself. If you have already run the simulation you will need
+to delete or rename the output files, as previously generated output can prevent the
```

Comment:
Should we give an example code to remove the files (e.g., using `rm`) for those who have little experience with shell?